.github/workflows/deploy.yml

```diff
@@ -0,0 +1,49 @@
+name: Test and Build React
```

Comment:
Can you update the workflow name for this one? Looks to be copied from the previous one

Reply:
Oh yes sure